app/src/main/res/layout/fragment_home.xml

```diff
@@ -20,6 +21,19 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+
```

Comment:
can you just add SwipeRefreshLayout here so add the refresh ability to user if he wants to get new channels